src/site/twitch.tv/modules/video-player/VideoPlayerModule.vue

```diff
@@ -0,0 +1,49 @@
+<template />
+
+<script setup lang="ts">
+import { declareModule } from "@/composable/useModule";
+import { declareConfig, useConfig } from "@/composable/useSettings";
+import { watchEffect } from "vue";
+
+const { markAsReady } = declareModule("video-player", {
+	name: "Video Player",
+	depends_on: [],
+});
+
+const pauseOnClickEnabled = useConfig<boolean>("channel.video_player.pause_on_click");
+
+const videoPlayerClickHandler = document.querySelector('div[data-a-target="player-overlay-click-handler"]');
```

Comment:
Please don't use DOM operations. We have built an array of React utilities capable of hooking Twitch's components, you can see those under `common/ReactHooks.ts` and `common/Reflection.ts`.

In order to hook to hook the relevant component, a predicate such as this might work (untested, only visually found this scrolling through the react tree)
```ts
(n) => n.props && n.props.mediaPlayerInstance && n.props.videoContainer
```